command/bridge/exit/exit.go

```diff
@@ -233,7 +233,9 @@ func createExitTxn(sender ethgo.Address, proof types.Proof) (*ethgo.Transaction,
 		return nil, nil, fmt.Errorf("failed to unmarshal exit event from JSON. Error: %w", err)
 	}
 
-	exitEventEncoded, err := polybft.ExitEventInputsABIType.Encode(exitEvent)
+	var exitEventAPI contractsapi.L2StateSyncedEvent
+
+	exitEventEncoded, err := exitEventAPI.Encode(exitEvent)
```

Comment:
This API looks a bit strange, should `Encode` and other methods be moved to a non-member method?

Reply:
We could move `Encode` function on events to non-member method, since it doesn't change the state of the object (in this case `L2StateSyncedEvent`). I just wanted to keep it tight to the corresponding stubs. But yeah, I can move it to non-member. Just keep in mind we would have `n` number of `Encode{NameOfEvent}` functions, where `n` is the number of event stubs we generate.

Reply:
this generated API doesn't looks neat at all, but it's ok for now but we can remove the object in the member method. `(o *Object) Method() {}` -> `(*Object) Method() {}` if `o` is not used in the member method

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1321/commits/ba6096ec656b5b8b1a27892a9f6f61a33f493f95